code/UMLEditor.py

```diff
@@ -21,6 +22,8 @@ def main():
     if len(sys.argv) == 1:
         print ("Running WebApp version of UMLEditor")
         print ("Open a browser and go to 'localhost:5000' to see the local webpage")
+        # Opens up GUI application in user's default browser
+        webbrowser.open('http://localhost:5000/')
         GUIMain.app.run(debug=True)
```

Comment:
Debug mode causes the 2 tabs because each time debug is updated, it re-runs all of the code. That's how they implemented updating the server when you save a file, so that you don't have to close and re-open the server yourself.